Makefile

```diff
@@ -0,0 +1,20 @@
+MODULE = gammapy
+
+CURDIR = $(shell pwd)
+
+build:
```

Comment:
Now this is confusing. Why is `make build` equivalent to `python setup.py develop` and not `python setup.py build`?
I'd say, let's just stick to the commands as they are in setup.py and make this consistent, even if their names are a bit cryptic at times.